<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1455</width>
    <height>787</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>641</width>
      <height>461</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_7">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <item>
        <widget class="QWidget" name="Header" native="true">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(255, 170, 127);</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLabel" name="CompName">
            <property name="font">
             <font>
              <pointsize>10</pointsize>
              <bold>true</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">color: rgb(255, 255, 255);</string>
            </property>
            <property name="text">
             <string>TATA ELXSI</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="TimeLabel">
            <property name="font">
             <font>
              <pointsize>10</pointsize>
              <bold>true</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">color: rgb(255, 255, 255);</string>
            </property>
            <property name="text">
             <string>Date</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="widget" native="true">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(255,255,240);</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QListWidget" name="listWidget">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="font">
               <font>
                <pointsize>15</pointsize>
                <bold>true</bold>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">QListWidget
{
border:none;

	background-color: rgb(245,222,179);
color:rgb(52,51,64);
min-width:300px;
max-width: 300px;
min-height:300px;
max-height:300px;
padding: 0px;
}</string>
              </property>
              <property name="verticalScrollBarPolicy">
               <enum>Qt::ScrollBarAsNeeded</enum>
              </property>
              <property name="sizeAdjustPolicy">
               <enum>QAbstractScrollArea::AdjustToContentsOnFirstShow</enum>
              </property>
              <property name="autoScrollMargin">
               <number>10</number>
              </property>
              <property name="iconSize">
               <size>
                <width>30</width>
                <height>30</height>
               </size>
              </property>
              <property name="verticalScrollMode">
               <enum>QAbstractItemView::ScrollPerPixel</enum>
              </property>
              <property name="horizontalScrollMode">
               <enum>QAbstractItemView::ScrollPerPixel</enum>
              </property>
              <property name="movement">
               <enum>QListView::Free</enum>
              </property>
              <property name="isWrapping" stdset="0">
               <bool>false</bool>
              </property>
              <property name="viewMode">
               <enum>QListView::ListMode</enum>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QWidget" name="MusicPlayer" native="true">
              <property name="styleSheet">
               <string notr="true">background-color: rgb(255,255,240);</string>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_3">
               <item>
                <layout class="QVBoxLayout" name="verticalLayout_2">
                 <item>
                  <widget class="QLabel" name="labelSong_name">
                   <property name="font">
                    <font>
                     <pointsize>13</pointsize>
                     <bold>true</bold>
                    </font>
                   </property>
                   <property name="styleSheet">
                    <string notr="true">background-color: rgb(255, 170, 127);</string>
                   </property>
                   <property name="text">
                    <string>Song_name</string>
                   </property>
                   <property name="alignment">
                    <set>Qt::AlignCenter</set>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QGraphicsView" name="Song_Image">
                   <property name="styleSheet">
                    <string notr="true">background-color: rgb(255, 255, 255);</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <layout class="QVBoxLayout" name="verticalLayout">
                   <item>
                    <widget class="QSlider" name="Audio_Seeker">
                     <property name="styleSheet">
                      <string notr="true">background-color: rgb(255, 170, 127);</string>
                     </property>
                     <property name="orientation">
                      <enum>Qt::Horizontal</enum>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <layout class="QHBoxLayout" name="horizontalLayout">
                     <item>
                      <widget class="QPushButton" name="pushButton_Rewind">
                       <property name="font">
                        <font>
                         <pointsize>22</pointsize>
                         <bold>true</bold>
                        </font>
                       </property>
                       <property name="styleSheet">
                        <string notr="true">QPushButton
{
border:none;
border-radius:25px;
background-color:;
background-color: rgb(255, 170, 127);
color:rgb(52,51,64);
min-width:50px;
max-width: 50px;
min-height:50px;
max-height:50px;
padding: 0px;
}
QPushButton:Hover
{
background-color:rgb(255,216,177);
}</string>
                       </property>
                       <property name="text">
                        <string/>
                       </property>
                       <property name="icon">
                        <iconset>
                         <normaloff>Imgs_Songs/backward-arrows-couple.png</normaloff>Imgs_Songs/backward-arrows-couple.png</iconset>
                       </property>
                       <property name="iconSize">
                        <size>
                         <width>20</width>
                         <height>20</height>
                        </size>
                       </property>
                      </widget>
                     </item>
                     <item>
                      <widget class="QPushButton" name="pushButton_Play">
                       <property name="font">
                        <font>
                         <pointsize>19</pointsize>
                         <bold>true</bold>
                        </font>
                       </property>
                       <property name="styleSheet">
                        <string notr="true">QPushButton
{
border:none;
border-radius:25px;
background-color: rgb(255, 170, 127);
color:rgb(52,51,64);
min-width:50px;
max-width: 50px;
min-height:50px;
max-height:50px;
padding: 0px;
}
QPushButton:Hover
{
background-color:rgb(255,216,177);
}</string>
                       </property>
                       <property name="text">
                        <string/>
                       </property>
                       <property name="icon">
                        <iconset>
                         <normaloff>Imgs_Songs/play-button-arrowhead.png</normaloff>Imgs_Songs/play-button-arrowhead.png</iconset>
                       </property>
                       <property name="iconSize">
                        <size>
                         <width>20</width>
                         <height>20</height>
                        </size>
                       </property>
                      </widget>
                     </item>
                     <item>
                      <widget class="QPushButton" name="pushButton_Forward">
                       <property name="font">
                        <font>
                         <pointsize>21</pointsize>
                         <bold>true</bold>
                        </font>
                       </property>
                       <property name="styleSheet">
                        <string notr="true">QPushButton
{
border:none;
border-radius:25px;
background-color:;
background-color: rgb(255, 170, 127);
color:rgb(52,51,64);
min-width:50px;
max-width: 50px;
min-height:50px;
max-height:50px;
padding: 0px;
}
QPushButton:Hover
{
background-color:rgb(255,216,177);
}</string>
                       </property>
                       <property name="text">
                        <string/>
                       </property>
                       <property name="icon">
                        <iconset>
                         <normaloff>Imgs_Songs/forward.png</normaloff>Imgs_Songs/forward.png</iconset>
                       </property>
                       <property name="iconSize">
                        <size>
                         <width>20</width>
                         <height>20</height>
                        </size>
                       </property>
                      </widget>
                     </item>
                    </layout>
                   </item>
                  </layout>
                 </item>
                </layout>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QWidget" name="widget_2" native="true">
       <property name="styleSheet">
        <string notr="true">background-color: rgb(255, 170, 127);</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_6">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QPushButton" name="pushButton_Music">
            <property name="font">
             <font>
              <pointsize>22</pointsize>
              <bold>true</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton
{
border:none;
border-radius:25px;
color:rgb(52,51,64);
min-width:50px;
max-width: 50px;
min-height:50px;
max-height:50px;
padding: 0px;
}
QPushButton:Hover
{
background-color:rgb(255,216,177);
}</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset>
              <normaloff>Imgs_Songs/musical-note.png</normaloff>Imgs_Songs/musical-note.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>30</width>
              <height>30</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_Navigation">
            <property name="font">
             <font>
              <pointsize>22</pointsize>
              <bold>true</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton
{
border:none;
border-radius:25px;
color:rgb(52,51,64);
min-width:50px;
max-width: 50px;
min-height:50px;
max-height:50px;
padding: 0px;
}
QPushButton:Hover
{
background-color:rgb(255,216,177);
}</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset>
              <normaloff>Imgs_Songs/compass (2).png</normaloff>Imgs_Songs/compass (2).png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>30</width>
              <height>30</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_call">
            <property name="font">
             <font>
              <pointsize>22</pointsize>
              <bold>true</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton
{
border:none;
border-radius:25px;

color:rgb(52,51,64);
min-width:50px;
max-width: 50px;
min-height:50px;
max-height:50px;
padding: 0px;
}
QPushButton:Hover
{
background-color:rgb(255,216,177);
}</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset>
              <normaloff>Imgs_Songs/phone-call.png</normaloff>Imgs_Songs/phone-call.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>30</width>
              <height>30</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_mainmenu">
            <property name="font">
             <font>
              <pointsize>22</pointsize>
              <bold>true</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton
{
border:none;
border-radius:25px;

color:rgb(52,51,64);
min-width:50px;
max-width: 50px;
min-height:50px;
max-height:50px;
padding: 0px;
}
QPushButton:Hover
{
background-color:rgb(255,216,177);
}</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset>
              <normaloff>Imgs_Songs/menu.png</normaloff>Imgs_Songs/menu.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>30</width>
              <height>30</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_Pic">
            <property name="font">
             <font>
              <pointsize>22</pointsize>
              <bold>true</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton
{
border:none;
border-radius:25px;

color:rgb(52,51,64);
min-width:50px;
max-width: 50px;
min-height:50px;
max-height:50px;
padding: 0px;
}
QPushButton:Hover
{
background-color:rgb(255,216,177);
}</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset>
              <normaloff>Imgs_Songs/google-photos.png</normaloff>Imgs_Songs/google-photos.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>30</width>
              <height>30</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_weather">
            <property name="font">
             <font>
              <pointsize>22</pointsize>
              <bold>true</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton
{
border:none;
border-radius:25px;

color:rgb(52,51,64);
min-width:50px;
max-width: 50px;
min-height:50px;
max-height:50px;
padding: 0px;
}
QPushButton:Hover
{
background-color:rgb(255,216,177);
}</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset>
              <normaloff>Imgs_Songs/cloudy-day.png</normaloff>Imgs_Songs/cloudy-day.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>30</width>
              <height>30</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_Driver">
            <property name="font">
             <font>
              <pointsize>22</pointsize>
              <bold>true</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton
{
border:none;
border-radius:25px;

color:rgb(52,51,64);
min-width:50px;
max-width: 50px;
min-height:50px;
max-height:50px;
padding: 0px;
}
QPushButton:Hover
{
background-color:rgb(255,216,177);
}</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset>
              <normaloff>Imgs_Songs/steering-wheel.png</normaloff>Imgs_Songs/steering-wheel.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>30</width>
              <height>30</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1455</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
